pineappl_py/src/boc.rs

```diff
@@ -274,6 +274,17 @@ impl PyChannel {
         }
     }
 
+    /// Get the factors for all the channels.
+    ///
+    /// Returns
+    /// -------
+    /// list(float) :
+    ///     list containing the factor values
+    #[must_use]
+    pub fn factors(&self) -> Vec<f64> {
+        self.entry.entry().iter().map(|(_, f)| *f).collect()
+    }
+
```

Comment:
Is this (convenience) method really needed? I'm always in favour of keeping APIs as small as possible (but you may disagree); one can also use `.into_array()` and then extract the factor from the Python side. AFAIU you use this method to check that the FK-table factors are correct, but this is now guaranteed.

Reply:
Yes, I suggest we don't add anything we don't need (yet), especially if the functionally is already there. If you remove them, we should be able to merge this quickly.